docs/README.rst

```diff
@@ -1,28 +1,42 @@
 tractor
 =======
-A `structured concurrent`_, async-native "`actor model`_" built on trio_ and multiprocessing_.
+The Python async-native multi-core system *you always wanted*.
+
 
 |gh_actions|
 |docs|
 
 .. _actor model: https://en.wikipedia.org/wiki/Actor_model
 .. _trio: https://github.com/python-trio/trio
-.. _multiprocessing: https://en.wikipedia.org/wiki/Multiprocessing
+.. _multi-processing: https://en.wikipedia.org/wiki/Multiprocessing
 .. _trionic: https://trio.readthedocs.io/en/latest/design.html#high-level-design-principles
 .. _async sandwich: https://trio.readthedocs.io/en/latest/tutorial.html#async-sandwich
 .. _structured concurrent: https://trio.discourse.group/t/concise-definition-of-structured-concurrency/228
 
 
-``tractor`` is an attempt to bring trionic_ `structured concurrency`_ to
-distributed multi-core Python; it aims to be the Python multi-processing
-framework *you always wanted*.
+``tractor`` is a `structured concurrent`_ "`actor model`_" built on trio_ and multi-processing_.
+
+It is an attempt to pair trionic_ `structured concurrency`_ with
+distributed Python. You can think of it as a ``trio``
+*-across-processes* or simply as an opinionated replacement for the
+stdlib's ``multiprocessing`` but built on async programming primitives
+from the ground up.
+
+Don't be scared off by this description. ``tractor`` **is just ``trio``**
```

Comment:
This bold mixed with default role stuff doesn't render properly on github

Reply:
Yah, need to definitely fix this plus stick in your other suggestions.

Thanks again for the thoroughness.

Reply:
I might actually delay branding updates until a new PR since this is getting a bit out of hand 😂